### PR TITLE
Draft of ZenIP 42203 for the creation of HorizenDAO

### PR DESCRIPTION
**Pull request by megaluck**

### CODE REVIEW

File: zenip-42203.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# ZenIP-42203
+
+    ZenIP: 42203
+    Title: HorizenDAO
+    Owners: Horizen Community Council, Erica Hamilton erica@horizen.io, Dean Steinbeck dean@horizenlabs.io, Gustavo Fialho gustavo@zensystem.io 
+    Status: Draft
+    Type: Consensus
```
Comment (cronicc) [inline zenip-42203.md]: This ZenIP doesn't fall within the `Consensus` type, the appropriate type would be `Process`.

Please review the ZenIP process specification [zenip-42000.md#zenip-types](https://github.com/HorizenOfficial/ZenIPs/blob/bf607b30ce62d9dbae46ec02861e8255e59bea8a/zenip-42000.md#zenip-types) and assign the correct type. Please leave the ZenIP number/filename blank for later assignment by a ZenIP editor.

**Review by cronicc — CHANGES_REQUESTED** [review]

File: zenip-42203.md (hunk 1 above)
Comment (megaluck) [reply zenip-42203.md]: Updated

**Review by cronicc — APPROVED** [review]

**Review by rosariopabst — APPROVED** [review]

**Comment by tarrenj** [comment]
Is this an iteration of the earlier IOHK proposal, or a totally new venture? https://iohk.io/en/research/library/papers/a-treasury-system-for-cryptocurrencies-enabling-better-collaborative-intelligence/

**Review by zain-cheng — APPROVED** [review]

**Comment by finpunk** [comment]
> Is this an iteration of the earlier IOHK proposal, or a totally new venture? https://iohk.io/en/research/library/papers/a-treasury-system-for-cryptocurrencies-enabling-better-collaborative-intelligence/

The proposal is related to the org structure, not the specific implementation. Basically, it's asking the community if we want to launch a DAO.

**Pull request merged**